index.js

```diff
@@ -56,14 +58,34 @@ function groups (groups, data, aggregations, thisArg, additionalArgs) {
   data = Array.isArray(data) ? data : data.features
   var args = [undefined, undefined, undefined].concat(additionalArgs)
 
+  var tree = rbush(data.length)
```

Comment:
Actually we are doing a small number of searches (~200) because we only do one for each group. In general I think that will be the case (`groups.length < data.length`) which favors this method